karabo_data/tests/make_examples.py

```diff
@@ -255,12 +255,14 @@ def make_spb_run(dir_path, raw=True):
                 [
                     XGM('SA1_XTD2_XGM/DOOCS/MAIN'),
                     XGM('SPB_XTD9_XGM/DOOCS/MAIN'),
+                    BaslerCamera('SPB_IRU_SIDEMIC_CAM', px_size=px_size),
```

Comment:
Please use a name that follow the naming convention as I mentioned before

Reply:
Ah yes, will do